src/components/Popups/TransactionPopup.tsx

```diff
@@ -22,22 +24,29 @@ function TransactionPopupContent({ tx, chainId }: { tx: TransactionDetails; chai
   const tokens = useCombinedActiveList()
   const activity = parseLocalActivity(tx, chainId, tokens)
   const { ENSName } = useENSName(activity?.otherAccount)
+  const theme = useTheme()
 
   if (!activity) return null
 
   const explorerUrl = getExplorerLink(chainId, tx.hash, ExplorerDataType.TRANSACTION)
 
+  const currencies = activity.currencies?.filter((c) => c) as Currency[]
+
   return (
     <PortfolioRow
       left={
         success ? (
           <Column>
-            <PortfolioLogo
-              chainId={chainId}
-              currencies={activity.currencies}
-              images={activity.logos}
-              accountAddress={activity.otherAccount}
-            />
+            {currencies.length > 0 ? (
```

Comment:
`PortfolioLogo` already has logic to deal with undefined currencies. I think this file can be left as-is.